app/api/emails.py

```
@@ -0,0 +1,182 @@
+
+
+async def get_authenticated_caller_id(
+    x_caller_id: str = Header(..., alias="X-Caller-Id"),
+) -> str:
+    return x_caller_id
+
+
+@router.post("/", response_model=EmailResponse, status_code=status.HTTP_202_ACCEPTED)
+async def send_email(
+    email_request: EmailRequest,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)